### PR TITLE
add >10s limit

### PR DESCRIPTION
**Pull request by Stanliest**
IMG_9023

### CODE REVIEW

File: frontend/src/screens/recordingModule.js [hunk 1]
```diff
@@ -52,16 +49,29 @@ class RecordingModule extends Component {
...
+    if (this.state.time < 10) {
+      Alert.alert(
+        "Warning",
+        "Please record more than 10 seconds.",
+        [{
+          text: 'OK', onPress: () => {
+            this.setState({
+              video: null,
+              recording: false,
+              time: 0
```
Comment (gucci-ninja) [inline frontend/src/screens/recordingModule.js]: the stopTimer function sets time to 0 already so u dont need this line

File: frontend/src/screens/recordingModule.js [hunk 2]
```diff
@@ -52,16 +49,29 @@ class RecordingModule extends Component {
...
+            });
+          }
+        }])
+    } else {
+      this.setState({
+        recording: false,
+        playback: true
+      }, () => { this.cam.stopRecording(); })
```
Comment (gucci-ninja) [inline frontend/src/screens/recordingModule.js]: this is only run in the else block, put it outside so it runs regardless

File: frontend/src/screens/recordingModule.js (hunk 1 above)
Comment (Stanliest) [reply frontend/src/screens/recordingModule.js]: True, you right

File: frontend/src/screens/recordingModule.js [hunk 3]
```diff
@@ -52,16 +49,29 @@ class RecordingModule extends Component {
...
-      this.cam.stopRecording();
-    });
+    if (this.state.time < 10) {
+      Alert.alert(
+        "Warning",
+        "Please record more than 10 seconds.",
+        [{
+          text: 'OK', onPress: () => {
+            this.setState({
+              video: null,
```
Comment (gucci-ninja) [inline frontend/src/screens/recordingModule.js]: it doesn't let me re-record if video is set to null :'(

**Pull request merged**